google/ggrc-core
# GGRC-4257 Disabled code editing in all modals

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [ ] GGRC-4569

# Issue description

The code property shouldn't be editable anymore, so we had to make it read-only. That's what did I solve here. 

# Steps to test the changes

1) Open Edit modal of any object;

Actual result: you can edit "Code" field;
Expected result: you cannot edit "Code" field

# Solution description

It's a pretty straightforward solution, I replaced all of Code input fields with a read-only span tags.

# Sanity checklist




# PR Review checklist
